.github/contributing.md

```diff
@@ -0,0 +1,97 @@
+<p align="center">
+  <img src="/assets/UNiDAYS_Logo.png" />
+</p>
+<br/>
+
+# UNiDAYS Contributors Guide.
+
+:star: Thanks for taking the time to contribute :star: 
+
+This document outlines the different ways you can contribute to the UNiDAYS repositories, guidelines on how to do it and some of our standardisations. Everything is open to improvement and all of these are guidelines. If you can think of something we can change or are missing in this doc - you can even open a pull request for making that kind of change.
+
+[Code of Conduct](CODE_OF_CONDUCT.md)
+
+[What should I know before I get started?](#what-should-i-know-before-i-get-started)
+
+[How Can I Contribute?](#how-can-i-contribute)
+  * [Reporting Bugs](#reporting-bugs)
+  * [Suggesting Enhancements](#suggesting-enhancements)
+  * [Labels For Issues](#labels-for-issues)
+  * [Pull Requests](#pull-requests)
+  * [Your First Code Contribution](#your-first-code-contribution)
+
+[Style Guides](#style-guides)
+  * [Git Commit Messages](#git-commit-messages)
+  * [Style Guide Files Style Guide](#style-guide-files-style-guide)
+
+## How Can I Contribute?
+
+### Reporting Bugs :bug:
+
+When issuing a bug report please include as many of details on [the bug report template](./ISSUE_TEMPLATE/bug_report.md) as possible. If you find a Closed issue that seems like it is the same thing that you're experiencing, open a new issue and include a link to the original issue in the body of your new one. Please label the any bug report issues with `bug`.
+
+### Suggesting Enhancements 🔎
+
+Enhancements may include completely new features and minor improvements to existing functionality. Following these guidelines helps maintainers and the community understand your suggestion and find related suggestions.
+
+First of all please do a search on the issues already present to see if it's an enhancement that hasn't previously been suggested. If it has, maybe join the discussion on the pre-existing issue. This is to help reduce duplication of issues. If it hasn't been raised previously, please include as many details as you can using [the feature request template](./ISSUE_TEMPLATE/feature_request.md). Label the raised feature request as `enhancement`
+
+### Labels For Issues
+
+| Label       | Purpose             |

```

Comment:
This table formatting is FUBAR, doesn't render correctly at least in preview.

![image](https://user-images.githubusercontent.com/6706510/44215934-a238bc80-a16b-11e8-851a-526696e8107e.png)

Reply:
done